View/FrequencyRelationships/AnalyticalFrequencyEditor.xaml

```diff
@@ -56,7 +56,7 @@
         <GroupBox x:Name="group_graphical" Header="Graphical" Grid.Row="4"  Visibility="Visible">
                      
         <Grid x:Name="grid_graphical" Grid.Row="4" >
-                <frequencyrelationships:GraphicalTableWithPlotControl  Grid.Row="1" Visibility="{Binding ElementName=GraphicalButton, Path=IsChecked, Converter={StaticResource VisibleIfTrueConverter}}" DataContext="{Binding MyGraphicalVM}"/>
+                <frequencyrelationships:GraphicalTableWithPlotControl  Grid.Row="1" Visibility="{Binding ElementName=GraphicalButton, Path=IsChecked, Converter={StaticResource VisibleIfTrueConverter}}" DataContext="{Binding GraphicalTableWithPlotVM}"/>
```

Comment:
You are binding to GraphicalButton but i don't see a GraphicalButton anywhere in FDA.

Reply:
yep. that was a copy paste error. got rid of it.